### PR TITLE
Feature/793 wiring components docs

### LINKED ISSUE #793: Update documentation about "wiring components together" to explain output data streams

### PR DESCRIPTION
**Pull request by tomaszguzialek**
Is that good enough? :)

Fixes #793

### CODE REVIEW

**Comment by tomaszguzialek** [comment]
I gave up with a nice continuous screenshot of the whole page...

1:
wiring_screen_001

2:
wiring_screen_002

3:
wiring_screen_003

**Comment by kaspersorensen** [comment]
I think we need to make it clear that there are three ways of wiring and each of them should have it's own little section.

 * Transformer output
 * Filter requirements
 * Output data streams

The text for these sections is almost there in the previous section, we just need to make sub sections

File: documentation/src/docbkx/chapter12-getting-started-desktop.xml [hunk 1]
```diff
@@ -281,6 +281,48 @@
 				job.
 			</para>
 		</tip>
+		<section id="wiring_components_output_data_streams">
+			<title>Output data streams</title>
+			<para>The "Link to..." option wires components together in the "main
+				flow". However, some components are able to produce additional
+				output
+				data streams. For example, the main feature of a Completeness
+				Analyzer is to
```
Comment (ClaudiaPHI) [inline documentation/src/docbkx/chapter12-getting-started-desktop.xml]: Can you add a link to the "OutputDataStream" documentation ?

Comment (ClaudiaPHI) [inline documentation/src/docbkx/chapter12-getting-started-desktop.xml]: Could you also add an image with "Completeness analyzer" output streams:complete and incomplete ?

Comment (tomaszguzialek) [inline documentation/src/docbkx/chapter12-getting-started-desktop.xml]: AFAIR we don't have any other chapter regarding output data streams. But I will consider a screenshot with Complete records and Incomplete records output data streams and jobs built around them.

**Comment by tomaszguzialek** [comment]
Added subsection titles.
Added an example job with output data streams at the end:
2015-10-19 10_19_01-wiring components together

**Comment by ClaudiaPHI** [comment]
:+1:

**Pull request merged**